tests/unit/amo/components/TestAddonAdminLinks.js

```diff
@@ -0,0 +1,134 @@
+import * as React from 'react';
+
+import AddonAdminLinks, {
+  AddonAdminLinksBase,
+} from 'amo/components/AddonAdminLinks';
+import {
+  ADDONS_CONTENTREVIEW,
+  ADDONS_EDIT,
+  ADDONS_POSTREVIEW,
+  ADMIN_TOOLS_VIEW,
+  THEMES_REVIEW,
+} from 'core/constants';
+import { createInternalAddon } from 'core/reducers/addons';
+import {
+  dispatchClientMetadata,
+  dispatchSignInActions,
+  fakeAddon,
+  fakeTheme,
+} from 'tests/unit/amo/helpers';
+import { fakeI18n, shallowUntilTarget } from 'tests/unit/helpers';
+
+describe(__filename, () => {
+  const { store } = dispatchClientMetadata();
```

Comment:
This is dangerous because it means the state of each test will leak into the others.

You need to reset the state before each test within `beforeEach(() => ...)` or move the initialization into `render()` or create it with a common helper or something else.

Reply:
Oops, good catch. Thanks!